### PR TITLE
Showcase more services

### PR DESCRIPTION
**Pull request by da-kami**
This fills up empty space and shows the vision better.

image

image

### CODE REVIEW

File: lib/models/service_model.dart [hunk 1]
```diff
@@ -1,29 +1,48 @@
...
-  static const icons = {Service.cfd: Icons.insights, Service.sportsbet: Icons.sports_football};
+  static const labels = {
+    Service.cfd: "CFD Trading",
+    Service.sportsbet: "Sports Bets",
+    Service.exchange: "Taro Exchange",
+    Service.savings: "Saving"
+  };
+  static const shortLabels = {
+    Service.cfd: "CFDs",
+    Service.sportsbet: "Bets",
```
Comment (bonomat) [inline lib/models/service_model.dart:34]: Nice idea :) 

Is it possible to disable them or show them in a lighter color do indicate they are disabled?

**Review by bonomat — APPROVED** [review]

File: lib/models/service_model.dart (hunk 1 above)
Comment (da-kami) [reply lib/models/service_model.dart:34]: How about adding "coming soon" screens instead? - then the button would be enabled but there's only a screen that says coming soon (potentially with some more words on what is planned...?) 
I imagine that could be interesting if anybody joins the demo.

**Comment by da-kami** [comment]
Note: One more commit coming shortly :)

**Comment by da-kami** [comment]
Example placeholder:

image

**Pull request merged**